packages/client/auth-core/src/contexts/authlogin.tsx

```diff
@@ -17,27 +17,29 @@ const AuthContext = createContext<AuthContextType>({
     jwt: null,
 });
 
-type AuthProviderParams = {
+export type AuthProviderParams = {
     apiKey: string;
     environment: CrossmintEnvironment;
     children: ReactNode;
+    embeddedWallets: {
+        type: "evm-smart-wallet";
+        defaultChain: "polygon-amoy" | "base-sepolia";
+        createOnLogin: "all-users" | "off";
+    };
+};
+
+const getJwtFromCookie = (): string | null => {
+    if (typeof document === "undefined") return null; // Check if we're on the client-side
```

Comment:
todo: reminder to check in with this

Reply:
I think it's needed, but should be opened in a separate PR imo